dotm_search.py

```diff
@@ -1,15 +1,50 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
+import os
+import argparse
+import zipfile
+parser = argparse.ArgumentParser()
+
 
 """
 Given a directory path, search all files in the path for a given text string
 within the 'word/document.xml' section of a MSWord .dotm file.
 """
-__author__ = "???"
+__author__ = "knmarvel"
+#   time spent on this project: 2 hrs
+
+
+def examine_files(dir, char):
+    files = {}
+    total_files_searched = 0
+    for filename in os.listdir(dir):
+        filepath = dir + "/" + filename
+        if zipfile.is_zipfile(filepath):
```

Comment:
the README said specifically to look only in the .dotm files. I am thinking it's possible that a given directory might have other zip files that aren't in .dotm format and might get accidentally counted/surveyed this way.

Reply:
Below you asked where office/word came from...that's actually what removes things that aren't .dotm files.